### DISCUSSION TITLE
Is there a way to extract image prompts from metadata like A1111 does?

### DISCUSSION THREAD

**Discussion by Uneternal** [lllyasviel/Fooocus#1702] [Q&A] [answered]
In my workflow I often used this function of A1111 where you just drag&drop a picture and it tells you the prompt and the image generation data. This is the only feature I really miss in Fooocus, or did I just overlook it?
Also please update the standard model to Juggernaut V7. I tried changing it in the config and it's also already in the checkpoint folder, but it still downloads V6.

**Comment by wisdombox44** [comment]
Watch from 5:39
https://www.youtube.com/watch?v=S4b5tQoUfBA&t=147s

**Comment by mashb1t** [comment, marked as answer]
@Uneternal there already is an ongoing discussion in https://github.com/lllyasviel/Fooocus/issues/241, feel free to have a look
